src/history/hs.c

```diff
@@ -1398,10 +1458,12 @@ __hs_delete_key_from_ts_int(
      * beginning.
      */
     if (exact < 0) {
-        while ((ret = hs_cursor->next(hs_cursor)) == 0) {
+        ret = __wt_hs_cursor_next(session, hs_cursor);
+        while (ret == 0) {
```

Comment:
The old style of having the function call in the while loop condition is more consistent with WiredTiger style.

Reply:
aedc9c4